GoogleDataTransport/GDTCORTests/Common/Categories/GDTCORUploadCoordinator+Testing.m

```diff
@@ -31,12 +31,16 @@ - (void)reset {
 
 - (void)setTimerInterval:(uint64_t)timerInterval {
   [self setValue:@(timerInterval) forKey:@"_timerInterval"];
-  dispatch_source_set_timer(self.timer, DISPATCH_TIME_NOW, timerInterval, self.timerLeeway);
+  if (self.timer) {
```

Comment:
Consider saving self.timer to a local variable to prevent future crashing bugs where it's nil'd between this check and the call to dispatch_source.

Reply:
Good call, thanks!